CombinatorialGames/Game/Birthday.lean

```diff
@@ -261,7 +261,15 @@ theorem mem_birthdayFinset_succ {x : IGame} {n : ℕ} : x ∈ birthdayFinset (n
 
 @[simp] theorem birthdayFinset_zero : birthdayFinset 0 = {0} := rfl
 
-@[simp] theorem card_birthdayFinset_one : (birthdayFinset 1).card = 4 := by simp [birthdayFinset]
+@[simp]
+theorem Set.empty_ne_singleton {α : Type*} (a : α) : ∅ ≠ ({a} : Set α) :=
```

Comment:
This should be upstreamed to Mathlib, alongside the analogous `Finset` theorem. I'd do it myself, but I haven't figured out how the new contribution system works...